lib/index.js

```diff
@@ -106,6 +130,7 @@ exports.Model = class SchwiftyModel extends Model {
 
     parseValidationError(err) {
```

Comment:
Not weird in a way I don't wanna do it, just a shower thought like using a word in it's own definition loL